api/src/services/user-db.js

```diff
@@ -6,7 +6,10 @@ const db = require('../db-nano'),
 // https://wiki.apache.org/couchdb/HTTP_database_API#Naming_and_Addressing
 const DB_NAME_BLACKLIST = /[^a-z0-9_$()+/-]/g;
 
-const readMapFunction = function(doc) {
+// Space added after function to make Function.toString() output consistent
+// across node versions: https://github.com/nodejs/node/issues/20355
+// We are currently testing the exact content of the map function in tests/unit/lib/user-db.js
+const readMapFunction = function (doc) {
```

Comment:
It'd be better to merge this separately. Also, consider changing the test to not have the space so the git log is clearer. Finally, maybe we shouldn't test the exact content in the unit test? We could either make sure the view is defined, or read the function from disk and compare the two.